src/ci/test-libfuzzer-dotnet.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -ex -o pipefail
+
+# Required environment variables:
+# - GOODBAD_DOTNET
+# - LIBFUZZER_DOTNET
+# - LIBFUZZER_DOTNET_LOADER
+# - SHARPFUZZ
+
+export GOODBAD_DLL='GoodBad/GoodBad.dll'
+
+TMP=$(mktemp -d)
+cd $TMP
+
+cp -r ${GOODBAD_DOTNET} GoodBad
+
+# Instrument DLL under test.
+${SHARPFUZZ} GoodBad/GoodBad.dll
+
+# Create seed and crash inputs.
+printf 'good' > good.txt
+printf 'bad!' > bad.txt
+
+# Test individual env vars.
+export LIBFUZZER_DOTNET_TARGET_ASSEMBLY="${GOODBAD_DLL}"
+export LIBFUZZER_DOTNET_TARGET_CLASS='GoodBad.Fuzzer'
+export LIBFUZZER_DOTNET_TARGET_METHOD='TestInput'
+
+${LIBFUZZER_DOTNET} --target_path=${LIBFUZZER_DOTNET_LOADER} good.txt
```

Comment:
This is not a nice or sustainable way to write integration test assertions. Please consider it a minimum-viable test / placeholder to support plumbing everything else together.

My preference is to follow up in a subsequent PR with a micro-framework that lets us:
- Easily test many more conditions / invocations
- Continue on any test case failure
- Report passes-fails individually